qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -12,7 +12,7 @@ class TestFscrypt(XFSTestsDev):
     def setup_xfsprogs_devs(self):
         self.install_xfsprogs = True
 
-    def test_fscrypt(self):
+    def check_mounter(self):
```

Comment:
Given it's functionality seems like it's more appropriate to name it as `is_client_kernel_mounted`.

Reply:
I will advise it.